packages/paste-website/src/pages/icon-system/index.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Usage Guidelines
+description: Overview on how to use icons in the context of Paste projects.
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import {Anchor} from '@twilio-paste/anchor';
+import {Button} from '@twilio-paste/button';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {LoadingIcon} from '@twilio-paste/icons/esm/LoadingIcon';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/icon-system/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+
+<Callout variant="warning">
```

Comment:
Remove the `warning` variant on this. It's just a heads up. Also move it to below the Usage Guidelines